delivery/components/TelemetryEntry.yaml

```diff
@@ -10,9 +10,16 @@ TelemetryEntry:
       type: integer
       format: int64
       description: Timestamp of the entry, in milliseconds elapsed since UNIX epoch.
+    mode:
```

Comment:
Let's go ahead and add the `request` field in here and reference the TelemetryRequest object you created.  I think I originally told you we could add it in later; but let's just do it now.